scripts/src/saforcertadmin/push_secrets.py

```diff
@@ -46,16 +50,29 @@ def get_repo_public_key(repo):
         logging.error(f"unexpected response getting repo public key : {response.status_code} : {response.reason}")
         sys.exit(1)
     response_json = response.json()
+
+    if prartifact.xRateLimit in r.headers:
+        print(f'[DEBUG] {prartifact.xRateLimit} : {r.headers[prartifact.xRateLimit]}')
+    if prartifact.xRateRemain in r.headers:
+        print(f'[DEBUG] {prartifact.xRateRemain}  : {r.headers[prartifact.xRateRemain]}')
+
+    if "message" in response_json:
+        print(f'[ERROR] getting public key: {response_json["message"]}')
+        sys.exit(1)
+
     return response_json['key_id'], response_json['key']
 
 def get_repo_secrets(repo):
     """Get the list of secret names of a github repository"""
     secret_names = []
     response = requests.get(f'https://api.github.com/repos/{repo}/actions/secrets', headers=headers)
     if response.status_code != 200:
-        logging.error(f"unexpected response getting secrets : {response.status_code} : {response.reason}")
+        logging.error(f"[ERROR] unexpected response getting public key : {response.status_code} : {response.reason}")
```

Comment:
I think error message was correct earlier.. no need to change here.

Reply:
Yes was trying to see if codeql was complaining about the word "secret" and forgot to change it back.